tools/wheel_resolver/__init__.py

```diff
@@ -71,7 +70,7 @@
 )
 @click_log.simple_verbosity_option(_LOGGER)
 def main(
-    url: typing.List[str],
+    url: typing.Tuple[str],
```

Comment:
Nit: If we're using python 3.9 onwards, you can just use `tuple` here, rather than `typing.Tuple`, btw.

```suggestion
    url: tuple[str],
```

Reply:
Alternatively, `Iterable` if you don't much mind what kind of iterable it is? `from collections.abc import Iterable`

Reply:
`python-rules` still supports 3.8

This changes is because click passes it as a `tuple` rather than `list` - it's a bad type hint from the original commit